parsec/backend/realm.py

```diff
@@ -78,6 +79,8 @@ class RealmStatus:
     def in_maintenance(self) -> bool:
         return bool(self.maintenance_type)
 
+class RealmStats:
```

Comment:
` @attr.s(slots=True, frozen=True, auto_attribs=True) ` is needed

without it, `size` is a class attribute (by opposition of a class instance attribute). This means you cannot do `RealmStats(size=42)`